powerhome/playbook
# [PLAY-259] Typescript Conversion: Home Address Street

#### Screens

![Screen Shot 2022-08-31 at 1 27 51 PM](https://user-images.githubusercontent.com/8194056/187730427-70e3921a-376e-47a0-b382-6f2e99d34002.png)

#### Breaking Changes

No breaking changes, but it modifies the typing of `homeId` prop from `number` to `string`. This had to be done because this prop is passed to the `hashtag` component, that only accepts `string`.

*do not merge* until [this PR ](https://github.com/powerhome/playbook/pull/2023) gets approved. They will need to be merged and its conflicts solved.

#### Runway Ticket URL

[PLAY-259](https://nitro.powerhrg.com/runway/backlog_items/PLAY-259)

#### How to test this

Start the server and check if all components are rendering correctly. There should be no console errors. Try using the kit with different props.

#### Checklist:
